tensorboard_plugin_3d/plugin.py

```diff
@@ -36,13 +40,26 @@ def __init__(self, context):
           context: A base_plugin.TBContext instance.
         """
         self._data_provider = context.data_provider
+        self._logdir = context.logdir
 
     def get_plugin_apps(self):
         return {
             "/client/*": self._serve_static_file,
             "/static/*": self._serve_static_file,
+            "/images": self._serve_image,
+            "/tags": self._serve_tags,
         }
 
+    @wrappers.Request.application
+    def _serve_image(self, request):
+        response = {'images': []}
+        for eis in self._images:
+            np_arr = tf.io.decode_image(eis).numpy()
+            if np_arr.ndim == 4:
+                np_arr = np_arr[:,:,:,0]
+            response['images'].append({'array': np_arr.tolist()})
```

Comment:
there are optimizations to make here in the future, but will work for now :rocket: